scripts/test-sim.sh

```diff
@@ -19,14 +19,14 @@ export STORJ_NETWORK_DIR=$TMP
 storj-sim -x network setup
 
 # run aws-cli tests
-storj-sim -x network test bash $SCRIPTDIR/test-sim-aws.sh
-storj-sim -x network test bash $SCRIPTDIR/test-uplink.sh
+storj-sim -x network test bash "$SCRIPTDIR"/test-sim-aws.sh
+storj-sim -x network test bash "$SCRIPTDIR"/test-uplink.sh
 storj-sim -x network destroy
 
 # setup the network with ipv6
 storj-sim -x --host "::1" network setup
 # aws-cli doesn't support gateway with ipv6 address, so change it to use localhost
-find $STORJ_NETWORK_DIR/gateway -type f -name config.yaml -exec sed -i "s/server.address: \"\[::1\]/server.address: \"127.0.0.1/" {} +
+find "$STORJ_NETWORK_DIR"/gateway -type f -name config.yaml -exec sed -i 's/server.address: "[::1]/server.address: "127.0.0.1/' '{}' +
```

Comment:
I think this change is the reason why tests are failing at the moment.

Reply:
Oh dang